frontend-v2/src/features/data/generateCSV.ts

```diff
@@ -0,0 +1,113 @@
+import Papa from 'papaparse';
+import {
+  DatasetRead,
+  ProtocolRead,
+  SubjectGroupListApiResponse,
+  UnitListApiResponse,
+  UnitRead
+} from '../../app/backendApi';
+import { parse } from 'path';
+
+type Row = { [key: string]: any };
+type Data = Row[];
+type SubjectBiomarker = {
+  subjectId: any;
+  subjectDatasetId: number | undefined;
+  time: any;
+  timeUnit: UnitRead | undefined;
+  value: any;
+  unit: UnitRead | undefined;
+  qname: string | undefined;
+  label: string;
+  id: number;
+};
+
+const HEADERS: string[] = [
+  'ID', 'Group', 'Time', 'Time_unit',
+  'Observation', 'Observation Unit', 'Observation ID', 'Observation Variable',
+  'Administration ID', 'Amount', 'Amt_unit', 'Amount Variable',
+  'Route', 'II', 'ADDL'
+];
```

Comment:
I think this covers all the columns that are needed in an export, but we can chaange it as needed.